index.html

```diff
@@ -8,112 +8,138 @@
 
+  <section id="background-main" class="main-page">
+  <div>
+    <div class="main-page-text">
     <h1 class="main-text">Hey There. I'm sonika</h1>
     <h2 class="profession">I'am a Software Developer</h2>
     <p class="main-description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
       don't hestiate to contact me.</p>
-
+    </div>
   <!-- social-media Icons -->
-
+     <div class="social-div">
+ 
       <ul class="social-media">
 
       <li>
-        <a class="social-icons" href="http://github.com/sonikak004">
+        <a class="social-icons sitop1" href="http://github.com/sonikak004">
       <img class="soc-ico" src="./images/icon0.png" alt="github">
         </a>
       </li>
 
       <li> 
-        <a class="social-icons" href="http://linkedin.com/sonikak004">
+        <a class="social-icons sitop2" href="http://linkedin.com/sonikak004">
       <img class="soc-ico" src="./images/icon1.png" alt="linkedin2" >
         </a>
       </li>
 
       <li>
-        <a class="social-icons" href="#">
+        <a class="social-icons sitop3" href="#">
       <img class="soc-ico" src="./images/icon2.png" alt="V" >
         </a>
       </li>
 
       <li> 
-        <a class="social-icons" href="http://twitter.com/sonikakumar">
+        <a class="social-icons sitop4" href="http://twitter.com/sonikakumar">
       <img class="soc-ico" src="./images/icon3.png" alt="twitter">
         </a>
       </li>
 
       <li>
-        <a class="social-icons" href="#">
+        <a class="social-icons sitop5" href="#">
       <img class="soc-ico" src="./images/icon4.png" alt="M">
          </a>
       </li>
     </ul>
+  </div>
+  </div>
+ 
     <!-- my recent works page -->
 
   <section class="second-page">
-    
+    <div class="recent-div">
+    <div class="second-page-div">
     <h2 class="second-heading">My Recent Works</h2>
-    <hr>
-
+    <hr class="page2-line">
+  </div>
     <!-- image  one for second page -->
 
     <a class="second-background">
-      <img src="./images/second-background.png" alt="background-image">
+      <img class="background2" src="./images/second-background.png" alt="background-image">
     </a>
 
     <!-- multi post stories section -->
+    <div class="page2-para">
+    <div class="recent-works-div">
 
     <h3 class="recent-work-heading">Multi-Post Stories</h3>
 
-    <p class="recent-work-para">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-
+    <p class="recent-work-para">A daily selection of privately personalized reads; no accounts or sign-ups required. This has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+    
+    </div>
     <!-- languages cards  -->
 
```

Comment:
- [ ] Good job so far :+1: , however to stick with the figma design as much as possible, I think we can improve the My works section, so currently please notice the Featured project i.e. your Multi-Post stories card and the rest of the regular protect cards are not aligned properly :point_down: 
![image](https://user-images.githubusercontent.com/97521557/230708736-a53f8c82-79e1-4f53-92e0-028c3331285f.png)

Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/230708745-db7b32b4-3f1c-4934-88c3-cb8278f77265.png)

please try to fix this

- [ ] [optional]  I see that you have used position for the styling of your elements, I would advise to use flex or Grid as they are responsive and crucial skills to master, using position will not give you much room for responsive and control over your elements whereas using flex or grid will give you a lot of control over your elements and they are responsive as well :+1: